images/egress-router/Dockerfile

```diff
@@ -1,8 +1,18 @@
 FROM golang:1.13 as builder
+
 WORKDIR /go/src/github.com/openshift/egress-router-cni
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
```

Comment:
I'm not an expert on go mod, but my gut feeling is that there should be a script within hack that updates vendored dependencies, and the Dockerfile should just copy whatever is in repo rather than running that within the Docker image build.
I'll defer to @dougbtv which has worked on other multus CNI image repos to comment.

Reply:
@rcarrillocruz has it just right...

You'll want to do a `go mod vendor` either by hand, or in a `./hack` script (and a `go mod tidy`).

This is primarily because in the downstream build, you can't do any build time downloads, so, you need to have everything in a `./vendor/` directory.